firedrake/interpolation.py

```diff
@@ -192,7 +192,12 @@ def make_interpolator(expr, V, subset, access):
                 # function space nodes on the source mesh. NOTE: argfs_map is
                 # allowed to be None when interpolating from a Real space, even
                 # in the trans-mesh case.
-                argfs_map = compose_map_and_cache(target_mesh.cell_parent_cell_map, argfs_map)
+                if source_mesh.extruded:
+                    # ExtrudedSet cannot be a map target so we need to build
+                    # this ourselves
+                    argfs_map = vom_cell_parent_node_map_extruded(target_mesh, argfs.cell_node_map())
```

Comment:
`argfs_map` is already defined as `argfs.cell_node_map()`. You don't need the function call here and can use the variable instead.